gramps_webapi/data/apispec.yaml

```diff
@@ -131,6 +131,37 @@ paths:
         422:
           description: "Unprocessable Entity: Invalid token."
 
```

Comment:
Thanks for adding this! Had forgotten it.

Reply:
Actually you know I should try to review all the error codes... we're probably missing a number of them on the endpoints.  Hmmm....

Reply:
Yeah don't merge this yet.  I will look those over later tonight when I have time.

Reply:
Hey shouldn't the /login and /refresh endpoints return 201 and not 200 as they are posts and respond with a new token?

Reply:
And then update password would be a put not a post and return 200 as it updates the password?  Actually refresh is more like an update too since it's changing the token?

Reply:
Hmm... since the login & refresh do not modify anything on the server side, I would say 200 is fine (by using JWT, there is no session handling on the server at all and no concept of users being logged in or not).

But you are right about using put and returning 200 for the password update!